CMakeLists.txt

```diff
@@ -59,17 +63,19 @@ set_target_properties(
   SOVERSION "${ADA_LIB_SOVERSION}"
   WINDOWS_EXPORT_ALL_SYMBOLS YES
 )
-
-include (TestBigEndian)
-TEST_BIG_ENDIAN(IS_BIG_ENDIAN)
-if(IS_BIG_ENDIAN)
-  message(STATUS "Big-endian system detected.")
+if(NOT EMSCRIPTEN)
```

Comment:
this is not needed if `CMAKE_MODULE_PATH` is not set

Reply:
Point taken but we don't need this big endian stuff anyhow.